config/initializers/devise.rb

```diff
@@ -225,7 +225,7 @@
   # ==> Configuration for :timeoutable
   # The time you want to timeout the user session without activity. After this
   # time the user will be asked for credentials again. Default is 30 minutes.
-  config.timeout_in = 1.hour
+  config.timeout_in = 30.minutes
```

Comment:
```suggestion
  config.timeout_in = 20.hours
```
See Chris's comment

Reply:
Cancelling this suggestion, let's merge 🚀

Reply:
Would have been nice to see the commit message updated with the rationale for 30 minutes as the ultimate choice 🙃

Reply:
Good call:
As per the discussion on https://docs.google.com/spreadsheets/d/15AiWew6boVL3tGa0Xw130p2UlajT1jRGKTQzYkthYnc/
The 20 hour requirement is AAA; our level of compliance is AA. Since W3C recommendation contradicts the OWASP (security body) recommendation, there is further discussion going on on the sidelines. In the meantime, we are going to meet the EOW deadline for the ITHC issue raised and consider if we want to meet AAA in the future.